extensions-contrib/kubernetes-overlord-extensions/src/main/java/org/apache/druid/k8s/overlord/KubernetesTaskRunner.java

```diff
@@ -106,7 +106,7 @@ public class KubernetesTaskRunner implements TaskLogStreamer, TaskRunner
   private final HttpClient httpClient;
   private final PeonLifecycleFactory peonLifecycleFactory;
   private final ServiceEmitter emitter;
-
+  private static String WORKER_CATEGORY = "_k8s_worker_category";
```

Comment:
Can you add a javadoc here explaining why this is the worker category.

I believe it is because the KubernetesTaskRunner does not support worker categories currently, so we are just using this name to report metrics since that is what the interface calls for.

`getTotalTaskSlotCount` uses the category `taskQueue`. Which one do you think we should standardize on?

Reply:
Yup, we can use this `WORKER_CATEGORY` as the naming convention is more aligned with other places.